src/lib/list/selection-list.ts

```diff
@@ -279,11 +279,11 @@ export class MatListOption extends _MatListOptionMixinBase
   inputs: ['disabled', 'disableRipple', 'tabIndex'],
   host: {
     'role': 'listbox',
-    '[tabIndex]': 'tabIndex',
     'class': 'mat-selection-list',
     '(focus)': 'focus()',
     '(blur)': '_onTouched()',
     '(keydown)': '_keydown($event)',
+    '[attr.tabindex]': 'disabled ? 0 : tabIndex',
```

Comment:
Shouldn't this be just `tabIndex`? Either way the list will be focusable, unless the consumer passed in -1, in which case this will make it focusable only when disabled.

Reply:
With `-1` the selection list won't be tabbable anymore, but it should be tabbable if disabled for a11y.

Reply:
What I meant was that with this setup, if the consumer set it to -1 explicitly, it would only be tabbable when it's disabled which seems wrong.